SonarQube.MSBuild.Tasks/AttachBuildWrapper.cs

```diff
@@ -23,13 +23,14 @@ public class AttachBuildWrapper : Task
         [DllImport("kernel32.dll")]
         private static extern IntPtr GetModuleHandle(string lpModuleName);
 
-        // See https://jira.sonarsource.com/browse/CPP-1458 for the specification of the files contained in the C++ static resource zip
-        private const string BuildWrapperExeName = "build-wrapper-win-x86-64.exe";
+        // See https://jira.sonarsource.com/browse/CPP-1458 for the specification of the files contained in the Cpp static resource zip
```

Comment:
Miscellaneous: changed "C++" in the comment to "Cpp" as the former was causing S128 to fire. I'm checking whether this is a known issue.